Makefile

```diff
@@ -48,3 +56,9 @@ logs: ## Show logs
 send-api-req-local: ## Send a request to the local running server
 	curl -X POST http://localhost:8080/events -H 'Content-Type: application/json' -H 'Authorization: Bearer cn389ncoiwuencr' --data-binary "@./fixtures/forwarder.json" 
 
+.PHONY:dumpmetabase
+dumpmetabase: ## dump the database from db container
+	docker-compose exec db /bin/sh \
+	-c 'mysqldump -uroot -proot --databases metabase' \
+	| grep -v "Using a password on the command line interface can be insecure" \
```

Comment:
A little hacky. Apparently an alternative is to set an env var with the password:
https://stackoverflow.com/a/24188878

Not sure if it works with `mysqldump`.

```
MYSQL_PWD=root mysqldump -uroot --databases metabase
```

Reply:
found that as well but didn't work as expected.